### ISSUE TITLE
Project scope

### ISSUE THREAD

**Issue by leecalcote** [codeKraken19/go-server#1]
I'm just curious as to how expansive the scope of this project is, @codeKraken19. I wonder if you might elaborate with just a simple sentence.

**Comment by codeKraken19** [comment]
@leecalcote  yeah sure , this is a small attempt  to create a http server in go using the standard library
